### PR TITLE
Upgrade/800/sas logical interconnect groups

### PR DESCRIPTION
**Pull request by sijeesh**
### Description
Upgraded SAS Logical Interconnect Groups resource to support OneView API version 800

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: hpOneView/resources/networking/sas_logical_interconnect_groups.py [hunk 1]
```diff
@@ -79,84 +80,4 @@ def get_all(self, start=0, count=-1, filter='', sort='', scope_uris=''):
...
-            timeout:
-                Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation
-                in OneView; it just stops waiting for its completion.
-
-        Returns:
-            bool: Indicates if the resource was deleted successfully.
-
-        """
-        return self._client.delete(resource, force=force, timeout=timeout)
+        return self._helper.get_all(start, count, filter=filter, sort=sort, scope_uris=scope_uris)
```
Comment (soodpr) [inline hpOneView/resources/networking/sas_logical_interconnect_groups.py]: As per spec, we have query param as well. Can you please check it?

Comment (sijeesh) [reply hpOneView/resources/networking/sas_logical_interconnect_groups.py]: This query param is not new in this version, its available from API version 300. will add it here.

**Comment by sijeesh** [comment]
> query param for /rest/sas-logical-interconnect-groups API is mentioned in the spec. Please check the OneView UI as well.

 As I mentioned earlier, when we upgrade SDK from one version to another we only look for the changes in the new version, so there will be chances of missing these kinds of minor things.

**Comment by sijeesh** [comment]
> query param for /rest/sas-logical-interconnect-groups API is mentioned in the spec. Please check the OneView UI as well.

Added query param, please have a look.

**Pull request merged**